### DISCUSSION TITLE
Exclude window in rofi

### DISCUSSION THREAD

**Discussion by Moanrisy** [davatorium/rofi#1922] [Show and tell]
I have 2 window that I didn't want to show in rofi -show window

Using matching regex and filter make this possible

`rofi -show window -show-icons -matching regex -filter '^(?!.*(?:Clock|hello-tauri))'`

The only problem is rofi regex didn't accept space, but I will just rebuild Clock in Task class into Clock-in-Task later.

![2023-11-22T08:26:15,710812549+07:00](https://github.com/davatorium/rofi/assets/17421945/9aa63a69-e287-4ab0-8ebe-acd8452fc2c1)

**Comment by DaveDavenport** [comment]
Disable tokenizing?

**Reply by Moanrisy** [reply]
What is the meaning of disable tokenizing?

In Google it mean
'Tokenisation is the process of breaking up a given text into units called tokens. '


Is my matching filter called tokenizing?

**Reply by DaveDavenport** [reply]
No, if you check the manual of rofi, you see it breaks up the query based on spaces.  This option is called tokenize and can be disabled by passing the right config option.

See the [matching](https://davatorium.github.io/rofi/1.7.5/rofi.1/#matching) section in the manpage.

try passing `-no-tokenize`, this might fix your issue with accepting spaces:
> The only problem is rofi regex didn't accept space,

**Reply by Moanrisy** [reply]
Perfect, 👌

I can include space in my filter
